packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -97,30 +97,40 @@ const defaultErrorTypeForNormalize = "genericError";
  * @returns A valid Fluid Error with any provided annotations applied
  * @param error - The error to normalize
  * @param annotations - Annotations to apply to the normalized error
+ * @param messagePrefix - String to add to beginning of error message
  */
 export function normalizeError(
     error: unknown,
     annotations: IFluidErrorAnnotations = {},
+    messagePrefix?: string,
```

Comment:
If we add this param, add it to `IFluidErrorAnnotations` please, rather than a new param.